content/en/docs/v3.5/install.md

```diff
@@ -9,86 +9,160 @@ minGoVers: 1.20
 
 Before installing etcd, see the following pages:
 
-- [Supported platforms][]
-- [Hardware recommendations][]
+- [Supported platforms](/docs/v3.5/op-guide/supported-platform)
```

Comment:
Here and other places where you are adding links in bracket. I believe you don't need to that and you can leave what was in the original file. The links at the bottom of the fule takes care of it.